### PR TITLE
feat: idnow sdk upgrade to 4.1.1

### PR DESCRIPTION
**Pull request by NazarYablonskiy**

### CODE REVIEW

**Comment by NazarYablonskiy** [comment]
@pradel @arifaydogmus bitte, review

File: android/build.gradle [hunk 1]
```diff
@@ -33,18 +27,16 @@ android {
...
+        url "https://raw.githubusercontent.com/idnow/de.idnow.android/master"
+    }
 }
 
 dependencies {
     implementation 'com.facebook.react:react-native:+'
 
-    implementation 'de.idnow.sdk:idnow-android-4.1.2@aar'
-    // transitive dependencies of the idnow SDK
-    implementation 'org.slf4j:slf4j-android:1.7.12'
```
Comment (pradel) [inline android/build.gradle]: @NazarYablonskiy this are still required in the IDNOW documentation, are we sure we want to remove them?

Comment (NazarYablonskiy) [reply android/build.gradle]: @pradel `transitive dependencies` are required only if we use idnow sdk packaged into `.aar` file

Comment (NazarYablonskiy) [reply android/build.gradle]: but we don't anymore ;)

**Pull request merged**